README.md

```diff
@@ -24,7 +24,7 @@ Typically, a Feature is used when returning a single Item object, and FeatureCol
 JSON array of Item entities).
 
 The API can be implemented in compliance with the *[OGC API - Features](http://docs.opengeospatial.org/is/17-069r3/17-069r3.html)* standard 
-(we'll use OAFeat for shorthand). In this case STAC API can be thought of as a specialized Features API 
+(OAFeat as a shorthand). In this case STAC API can be thought of as a specialized Features API 

```

Comment:
Not sure, but this sounds weird to me due to the verb missing. Maybe make it easier to read for non-natives?
Or is it meant to be:
```suggestion
(OAFeat is a shorthand). In this case STAC API can be thought of as a specialized Features API 

```